src/app.ts

```diff
@@ -1,5 +1,19 @@
+#!/usr/bin/env node
+
+// first line is mandatory in JS, not sure about TS
+
+// try to run this with
+// node out/app.js -w 0:99 -d localhost:1234 -a db:pwd -b acc:container -k blob-key -t 1 -c crawl
+// node out/app.js -h
+
 import {ProcessingManager} from "./processing-manager";
 import {Worker} from "./worker";
+import {CLI} from "./cli";
+
+
+CLI.initCLI();
+CLI.logParms(); // will hide sensitive parameters, never log CLI.parameters directly!
```

Comment:
Are you sure that forked workers **don't** get the same command line arguments as the master? When running `node out/app.js -t 7` both master and workers change the number of threads to 7 (instead of default 4). Could you test it on your machine? 

If it works, there is no need to change anything. Each worker will parse arguments on it's own.